src/Illuminate/Database/Query/Grammars/SqlServerGrammar.php

```diff
@@ -200,6 +204,20 @@ public function compileTruncate(Builder $query)
     }
 
     /**
+     * Compile an exists statement into SQL.
+     *
+     * @param \Illuminate\Database\Query\Builder $query
+     *
```

Comment:
@GrahamCampbell This doc is a copy-paste from the base grammar. So we should fix it there too? :)